src/Components/test/testassets/TestServer/ServerStartup.cs

```diff
@@ -19,7 +19,7 @@ public ServerStartup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
-            services.AddServerSideBlazor(options => options.DetailedErrors = true);
+            services.AddServerSideBlazor();
```

Comment:
Out of interest, why is this change required?

Reply:
This option controls how some exceptions are logged: https://github.com/dotnet/aspnetcore/blob/ec2c7b478769c68870ec860592be27cc8920519b/src/Components/Server/src/Circuits/RemoteJSRuntime.cs#L43-L56
and it turns out some tests check for this behaviour: https://github.com/dotnet/aspnetcore/blob/9a1810c1dbe432fc7bc7e8bc68fa22ab787c0452/src/Components/test/E2ETest/ServerExecutionTests/ServerInteropTestDefaultExceptionsBehavior.cs#L66-L68

given there were several tests that seemed to be dependent on this options, I just removed it. It's possible that you might consider changing the log messages in the tests instead.

Reply:
This option affects the exception messages that are returned to the browser in JS interop calls. It doesn't only affect what gets logged to the browser console, it affects the exceptions that can be caught by JS code. The test code looks at the exceptions caught by JS code.

So I'm unclear on why this needs to be changed. I thought the only issue was with reading the browser console logs with the new version of Selenium, but the test in question (`DotNetExceptionDetailsAreNotLoggedByDefault`) wasn't doing that anyway.

> It's possible that you might consider changing the log messages in the tests instead.

Could you clarify what you're suggesting?

Reply:
No this has nothing to do with the browser console logs. Here the issue is that the test validates the exception message which is controlled by `DetailedErrors`. During the time we turned off e2etests, this option was modified to be true and therefore affected the exception message. The tests weren't updated in response to setting `DetailedErrors` to true and we saw failures when I re-enabled the tests. I opted to set `DetailedError` back to what it was before but the suggestion is that we can modify the exception message we verify to match the scenario where `DetailedError` is true as an alternative fix.

Reply:
That makes sense! Thanks for explaining. All LGTM here then!